client/state/themes/selectors.js

```diff
@@ -574,6 +574,18 @@ export function isThemePremium( state, themeId ) {
 }
 
 /**
+ * Whether a WPCOM theme given by its ID belongs to the Premium Squared bundle.
+ *
+ * @param  {Object} state   Global state tree
+ * @param  {Object} themeId Theme ID
+ * @return {Boolean}        True if the theme is in the Premium Squared bundle
+ */
+export function isPremiumSquaredTheme( state, themeId ) {
+	const theme = getTheme( state, 'wpcom', themeId );
+	return theme && includes( PREMIUM_SQUARED_THEMES, theme.stylesheet );
```

Comment:
> I don't think this would be a good idea, since a theme ID may conflict with a free theme. They may not conflict at this very moment (I haven't checked in depth though), but they might in the future, so this is not a solid and future-proof implementation.

If that was the case, we'd have to throw out most code of the showcase 😅  -- we rely on `themeId` being a reliable, well, ID, most everywhere. 😄 In other words: If you decide to stick with a static list, I think you can switch to unprefixed IDs without worrying too much. (I think there are lots of other places in the WP.com codebase that rely on the same premise, too.)

> These lists may rarely change in the future

Well it'll need to change everytime a new A8C or Woo premium theme is added, isn't it? Just sayin' 😄